Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/bpp_online/eval.py", line 9, in <module>
    from gpt import priority_v2 as priority
  File "/home/dokhanhnam1199/QD/problems/bpp_online/gpt.py", line 1, in <module>
    def priority_v2(item: float, bins_remain_cap: np.ndarray, epsilon: float = 0.628864920973181, tightness_weight: float = 0.29308373534145926, utilization_weight: float = 0.7459135811918961) -> np.ndarray:
NameError: name 'np' is not defined
Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/bpp_online/SLOC.py", line 7, in <module>
    from gpt import priority_v2 as priority
  File "/home/dokhanhnam1199/QD/problems/bpp_online/gpt.py", line 1, in <module>
    def priority_v2(item: float, bins_remain_cap: np.ndarray, epsilon: float = 0.628864920973181, tightness_weight: float = 0.29308373534145926, utilization_weight: float = 0.7459135811918961) -> np.ndarray:
NameError: name 'np' is not defined
Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/bpp_online/cyclomatic_complexity.py", line 7, in <module>
    from gpt import priority_v2 as priority
  File "/home/dokhanhnam1199/QD/problems/bpp_online/gpt.py", line 1, in <module>
    def priority_v2(item: float, bins_remain_cap: np.ndarray, epsilon: float = 0.628864920973181, tightness_weight: float = 0.29308373534145926, utilization_weight: float = 0.7459135811918961) -> np.ndarray:
NameError: name 'np' is not defined
